mmv1/templates/terraform/encoders/compute_backend_bucket.go.erb

```diff
@@ -0,0 +1,35 @@
+c, cdnPolicyOk := d.GetOk("cdn_policy")
+
+if !cdnPolicyOk {
+  return obj, nil
+}
+
+cdnPolicies := c.([]interface{})
+
+if len(cdnPolicies) == 0 {
+  return obj, nil
+}
+
+cdnPolicy := cdnPolicies[0].(map[string]interface{})
+
+cacheMode := cdnPolicy["cache_mode"].(string)
```

Comment:
shouldn't cacheMode be coming from `obj` not from the existing state? Default from `obj` and fallback from state?

What if someone is changing the cacheMode to `USE_ORIGIN_HEADERS`

We only want to remove these